ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1473,7 +1473,6 @@ presubmits:
     labels:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-installer-master-e2e-metal
-    optional: true
```

Comment:
We probably want to keep this for now

Reply:
But remove this: https://github.com/akiselev1/release/blob/enable-ci-always-run/ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml#L1550

There's code here that runs if any baremetal file changed, but I think we're ready to always turn it on

Reply:
Changed optional to false in both jobs and removed run_if_changed for installer.